homeassistant/components/tuya/switch.py

```diff
@@ -673,9 +690,11 @@
     ),
     # Smart Water Timer
     "sfkzq": (
-        SwitchEntityDescription(
+        TuyaDeprecatedSwitchEntityDescription(
             key=DPCode.SWITCH,
             translation_key="switch",
+            deprecated="replaced by a valve entity",
```

Comment:
We can't combine strings like this; it won't work with translations.

Reply:
> We can't combine strings like this; it won't work with translations.

I've updated the string to be more precise